Dockerfile.tumbleweed

```diff
@@ -9,16 +9,15 @@ FROM opensuse/${BUILD_FLAVOR}
 # the NON-OSS repo is not needed, save the network bandwidth and some time (~5 seconds) for each refresh
 RUN zypper mr -d repo-non-oss
 
-
 # we need to install Ruby first to define the %{rb_ver} RPM macro
 # see https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run
 # https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/build-cache
 # why we need "zypper clean -a" at the end
-RUN zypper --non-interactive in --no-recommends --force-resolution ruby && zypper clean -a
+RUN zypper --non-interactive in --no-recommends --force-resolution util-linux && zypper clean -a
 
 # add the YaST repository - we need rubygem-coveralls-lcov
 RUN zypper ar -f http://download.opensuse.org/repositories/YaST:/Head/openSUSE_Tumbleweed/ yast_tw
-RUN zypper ar -f http://download.opensuse.org/repositories/YaST:/Head/openSUSE_Leap_15.1_Update/ yast_lp
+RUN zypper ar -f http://download.opensuse.org/repositories/YaST:/Head/openSUSE_Leap_15.2_Update/ yast_lp
```

Comment:
Um, we have "openSUSE_Leap_15.2" target only (which already points to openSUSE:Leap:15.2:Update), see https://build.opensuse.org/repositories/YaST:Head